src/plugins/mer/merdeploysteps.cpp

```diff
@@ -259,22 +259,27 @@ bool MerProcessStep::init(InitOptions options)
     const FilePath projectDirectory = isShadowBuild
         ? bc->rawBuildDirectory()
         : project()->projectDirectory();
-    const FilePath toolsPath = engine->buildTarget(target).toolsPath;
-    CommandLine deployCommand(toolsPath.pathAppended(Sfdk::Constants::WRAPPER_DEPLOY));
-    deployCommand.addArgs(arguments(), CommandLine::Raw);
-
-    ProcessParameters *pp = processParameters();
+    // FIXME needed?
```

Comment:
So.. is it needed? Just pointing this out in case this is something you intended to look at before merging.

Reply:
That's indeed an open issue that I didn't found crucial to deal with immediately, so I left the comment just as a reminder for future. Thanks.